source/adminguide/accounts.rst

```diff
@@ -366,15 +366,15 @@ the user are used.
        #. CloudStack searches for it in LDAP by the configured
           ``ldap.username.attribute``.
 
-       #. If an LDAP user is found is found, CloudStack does a bind
+       #. If an LDAP user is found, CloudStack does a bind
```

Comment:
@Spaceman1984 can you remove this changes if not part of this PR ?

Reply:
I think fixing some stuff like this is fine, otherwise, no one will be fixing this explicitly :p

Reply:
Done